### PR TITLE
Add section about running the site in production-mode

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
Now that the pattern of using the environment as feature-flags to toggle certain pieces of site functionality is being used more often, coupled with the amount of work that will take place during the brand-consolidation transition, I thought it would be valuable to add information to our readme about how to locally start a webserver running a production build of the website, and also how to execute Nightwatch tests using that build.

## Testing done
- Ran the commands on my computer to make sure they were working okay

## Acceptance criteria
- [ ] Information makes sense to other engineers

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]
Noted some typos, but looks ok otherwise.

**Comment by kfrz** [comment]
Nice, @ncksllvn, this information is great to have up front. Ran through, suggested steps work as expected. 

LGTM!

File: README.md [hunk 1]
```diff
@@ -240,13 +252,26 @@ and you should see:
...
+#### Executing Nightwatch Tests For a Certain Environment
+A webserver is required for Nightwatch to execute. Locally, you will likely execute E2E tests while running the local development server via `npm run watch`, followed by `npm run test:e2e`.
+
+However, it is sometimes useful to execute E2E tests for a certain build-type, which is done by first running a build for that build-type, then executing the tests with the build-type passed as an argument.
+
+For example, to execute the E2E tests for Production:
+
+1. `NODE_ENV=production npm run build -- --buildtype=production`
+2. `BUILDTYPE=production npm run test:e2e`
+    - The Nightwatch startup script will see that port `3001` is not blocked (as it would be by the watch-task), and will start a [static webserver](#static-webserver) for the production build.
```
Comment (cvalarida) [inline README.md]: If we are running `yarn watch`, will it still connect to that if the `BUILDTYPE=production`?

**Review by cvalarida — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ncksllvn) [reply README.md]: Yeah, it always starts the static webserver at port 3001, so if that port is in use, it assumes that it must be a version of vets.gov.

**Review by kfrz — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**